### PR TITLE
Add how methods are versioned to the help center

### PR DESCRIPTION
**Pull request by jhmjam**
This PR covers the work needed to add Steve's version policy information

### CODE REVIEW

**Review by sanjeevz3009 — APPROVED** [review]
Looks good =)

File: content/help_centre/version-methods.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+The Statistical Methods Library (SML) and the associated methods are versioned using semantic versioning comprised of a major, minor and patch release version separated by full stops (e.g 3.5.4). 
+
+There are separate releases of the libraries based on the method implementation, python/pandas or python/pySpark, and each library uses semantic versioning.
+
+The major version is incremented (and minor and patch releases reset to zero) when backward incompatible changes are made (e.g renaming a pre-existing method function name).
+
+The minor version is incremented (and the patch version reset to zero) when new features are added without breaking backward compatibility (e.g addition of a new statistical method)
+
+The patch release version is incremented when bugfixes are implemented.
```
Comment (gibbardsteve) [inline content/help_centre/version-methods.md:9]: Could you make this line: The patch release version is incremented when backward compatible bugfixes are implemented.

It was a comment from Ian Dash when I shared the text with him.

File: content/help_centre/version-methods.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Consult the [SML python/pySpark release history in Github](https://github.com/ONSdigital/statistical-methods-library/releases) for the current and past releases for the python/pySpark methods and the [associated python/pySpark release notes](https://github.com/ONSdigital/statistical-methods-library/tree/main/docs/release-notes) to understand the changes between releases.
+
+Consult the [SML python/pandas release history in Github](https://github.com/ONSdigital/sml-python-small/releases) for the current and past releases for the python/pandas methods and the [associated python/pandas release notes](https://github.com/ONSdigital/sml-python-small/tree/main/docs/release-notes) to understand the changes between releases.
```
Comment (JasonBellONS) [inline content/help_centre/version-methods.md:13]: Not sure on standard for accessibility, do hyperlinks have to match to the link itself or do they just need to give a good description of what they link to?

Comment (jhmjam) [reply content/help_centre/version-methods.md:13]: Not sure but this was written by Steve :)

File: content/help_centre/version-methods.md (hunk 1 above)
Comment (jhmjam) [reply content/help_centre/version-methods.md:9]: Sure thing :)

File: content/help_centre/version-methods.md (hunk 2 above)
Comment (JasonBellONS) [reply content/help_centre/version-methods.md:13]: Ah ok ill follow this up with steve separately, thanks

**Review by JasonBellONS — APPROVED** [review]
All looks good to me

**Review by gibbardsteve — APPROVED** [review]
Looks good, we do need to wait for Fleur before we merge this though.  We also need to make sure the PR for the creation of the release notes is actioned at the same time.

**Pull request merged**